libs/workspace-manager/src/lib/workspace-manager/features/connect-workspace/connect-workspace.component.html

```diff
@@ -1 +1,12 @@
-<p>connect-workspace works!</p>
+<form
+  [formGroup]="form"
+  class="connect-workspace-form"
+  (ngSubmit)="connectWorkspace()"
+>
+  <h2 class="mat-title">Connect your Angular workspace</h2>
+  <mat-form-field class="workspace-path-field" appearance="outline">
```

Comment:
we don't have some UI lib to wrap all the material components ?

Reply:
No because we're using the vanilla material library, no customization needed